config/template.js

```diff
@@ -74,8 +71,7 @@ const SCHEMA_TEMPLATE = Joi.object().keys({
         .required()
         .or('image', 'template')
         .or('steps', 'template'),
-    images: TEMPLATE_IMAGES,
-    parameters: SCHEMA_JOB_PARAMETERS
```

Comment:
can users not put parameters in template?

Reply:
yes, they can. It goes under `config` similar to `environment`, `settings` and other attributes that are allowed on the job.